libcxx/test/std/re/re.alg/re.alg.match/awk.locale.pass.cpp

```diff
@@ -18,7 +18,7 @@
 
 // TODO: investigation needed
 // TODO(netbsd): incomplete support for locales
-// XFAIL: target={{.*}}-linux-gnu{{.*}}, netbsd, freebsd
+// XFAIL: target={{.*}}-linux-gnu{{.*}}, {{.*}}-amazon-linux{{.*}}, netbsd, freebsd
```

Comment:
IMO having it on a separate line makes it easier to read, and avoids mistakes.
```suggestion
// XFAIL: target={{.*}}-linux-gnu{{.*}}, netbsd, freebsd
// XFAIL: target={{.*}}-amazon-linux{{.*}}
```

Reply:
Changed according to your suggestion.